doc/api/modules.md

```diff
@@ -850,6 +850,21 @@ by the [module wrapper][]. To access it, require the `Module` module:
 const builtin = require('module').builtinModules;
 ```
 
+### module.makeRequire(filename)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filename` {string|URL} Filename for relative importing
+* Returns: {[`require`][]} Require function
+
+<!-- eslint-skip -->
+```js
+// create a require function which will resolve relative
+// to the current ES Module file
+const require = module.makeRequire(new URL(import.meta.url));
```

Comment:
`module` -> `require('module')` ?
(in sync with the example above and to not mix with module instances)